### DISCUSSION TITLE
Canadian tv channel to streamlink

### DISCUSSION THREAD

**Discussion by Sphinxroot** [iptv-org/community#243] [Show and tell]
there a couple channel not 24/7  but they have token 
 @LaneSh4d0w  could be possible to add them to your streamlink server ? please and thank :)

https://cfjctoday.com/live/
https://dcunilive72-lh.akamaihd.net/i/dclive_1@800677/master.m3u8 + token

https://www.cheknews.ca/live-news-british-columbia/
https://zypelive-lh.akamaihd.net/i/default_1@381849/master.m3u8 + token

https://www.chch.com/live/
https://flowcaster1-sh.cdn.clearcable.net/CHCH-OTT/smil:CHCH-OTT.smil/playlist.m3u8 + token

https://cable14now.com/live-streams/video/?videoId=104
https://flowcaster1-cl.cdn.clearcable.net/C14/smil:C14.smil/playlist.m3u8 + token

https://cable14now.com/live-streams/video/?videoId=592
https://flowcaster1-cl.cdn.clearcable.net/C14LS2/smil:C14LS2.smil/playlist.m3u8 + token

https://www.citytv.com/video/live/ DASH Stream + token

**Comment by BellezaEmporium** [comment]
Sure, let me check those.

**Comment by BellezaEmporium** [comment]
@Sphinxroot 

Tokens are IP-locked *
Webpage can't be accessed on my country ^
Geolock $

https://cfjctoday.com/live/ -> https://iptv-all.lanesh4d0w.codes/canada/cfjc

https://www.cheknews.ca/live-news-british-columbia/ -> https://iptv-all.lanesh4d0w.codes/canada/chek-news

https://www.chch.com/live/ ^

https://cable14now.com/live-streams/video/?videoId=104 -> https://iptv-all.lanesh4d0w.codes/canada/cable14 *

https://cable14now.com/live-streams/video/?videoId=592 -> https://iptv-all.lanesh4d0w.codes/canada/cable14-ls2 *

https://www.citytv.com/video/live/ -> https://iptv-all.lanesh4d0w.codes/canada/citytv-toronto $

**Reply by Dum4G** [reply]
Can confirm Chek news works here without VPN
cfjc -> An error occurred while processing your request.

**Comment by Sphinxroot** [comment]
thanks @LaneSh4d0w  :)

CFJC is offline for some reason from site

chek work  100%

cable 14 1 and 2 dont work

city dont work 

chch u need a vpn or the stream it self its geo locked and cant be restreamed ?

**Reply by BellezaEmporium** [reply]
Cable14 1 and 2 seems that the tokens are generated per IP, and so I'll need to use a proxy that everyone should use (like the Algerian channels)

City is geo-locked

For CHCH, the live stream webpage only is protected and needs a Canadian IP to get in there. I'll try getting it via a proxy.
